includes/integrations/maxmind-geolocation/class-wc-integration-maxmind-geolocation-database.php

```diff
@@ -0,0 +1,115 @@
+<?php
+/**
+ * WC Integration MaxMind Geolocation Database
+ *
+ * @version 3.9.0
+ * @package WooCommerce/Integrations
+ */
+
+defined( 'ABSPATH' ) || exit;
+
+/**
+ * WC Integration MaxMind Geolocation Database
+ *
+ * @version 3.9.0
+ */
+class WC_Integration_MaxMind_Geolocation_Database {
+
+	/**
+	 * The name of the MaxMind database to utilize.
+	 */
+	const DATABASE = 'GeoLite2-Country';
+
+	/**
+	 * The extension for the MaxMind database.
+	 */
+	const DATABASE_EXTENSION = '.mmdb';
+
+	/**
+	 * Fetches the path that the database should be stored.
+	 *
+	 * @return string The local database path.
+	 */
+	public static function get_database_path() {
+		$database_path = WP_CONTENT_DIR . '/uploads/' . self::DATABASE . self::DATABASE_EXTENSION;
```

Comment:
Hi, I think the database should be stored in a non-public directory to comply with the [Site License](https://www.maxmind.com/en/site-license-overview).